worker/src/lib/dbLogger/DBLoggable.ts

```diff
@@ -179,7 +182,16 @@ export async function dbLoggableRequestFromAsyncLogModel(
               asyncLogModel.timing.startTime.milliseconds
           )
         : new Date(),
-      bodyText: JSON.stringify(asyncLogModel.providerRequest.json),
+      // TEMP HACK
+      body: new ReadableStream({
```

Comment:
just for async logging. actually we can move this to be...
```suggestion
      body: JSON.stringify(asyncLogModel.providerRequest.json),
```

Reply:
@claude can you make this change?